src/main/resources/templates/db_scripts/schema.json

```diff
@@ -8,7 +8,7 @@
       "populateJsonWithId": true,
       "ginIndex": [
         {
-          "fieldName": "text",
+          "fieldName": "content",
```

Comment:
I don't think we need a ginIndex on content here -> probably a likeIndex on note_title would be sufficient since users are going to search by note title?

Reply:
sure -- make sense -- I'm assuming we will need to make some more adjustments as well once the GET Collections method is implemented

Reply:
Revised with like index on title -- question out on if this field should be unique